spritzle/core.py

```diff
@@ -104,13 +95,15 @@ def __init__(self, config, state_dir=None):
         self.session = lt.session(settings)
         await self.load_session_state()
         await self.alert.start(self.session)
-        await self.load_resume_data()
+        await self.resume_data.load()
```

Comment:
I think we could probably just have resume_data.start() call load() unless there is some scenario where we would start it without first loading the resume data.